modules/installation-special-config-raid.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-raid_{context}"]
+== Configuring a RAID-enabled data volume
+
+You can enable software RAID partitioning to provide an external data volume. {product-title} supports RAID 0, RAID 1, RAID 4, RAID 5, RAID 6, and RAID 10 for software backup, data protection, and fault tolerance. See "About disk mirroring" for more details.
+
+. Create a Butane config that uses software RAID to configure partitioning for two or more disks.
+
+* To configure mirrored boot disks that use RAID 1, create a `$HOME/clusterconfig/raid1-storage.bu` file, for example:
```

Comment:
The point here is the data partition, not the boot disk mirroring.  Maybe something like `To configure a data volume with RAID 1 on the same disks used for a mirrored boot disk, create...`?

Reply:
That makes sense, thanks.